src/defaultsettings.cpp

```diff
@@ -490,11 +539,7 @@ void set_default_settings()
 	settings->setDefault("touch_use_crosshair", "false");
 	settings->setDefault("fixed_virtual_joystick", "false");
 	settings->setDefault("virtual_joystick_triggers_aux1", "false");
-#ifdef ENABLE_TOUCH
-	settings->setDefault("clickable_chat_weblinks", "false");
-#else
-	settings->setDefault("clickable_chat_weblinks", "true");
-#endif
+	settings->setDefault("clickable_chat_weblinks", bool_to_cstr(!has_touch));
```

Comment:
Actually, is there any reason to disable this on touchscreens? I can't notice a difference.

I'm asking because if you have a laptop which also has a touchscreen, it's probably undesirable for links to be unclickable by default.

Reply:
Switched to `true` by default. Thanks!